.idea/compiler.xml

```diff
@@ -2,6 +2,7 @@
 <project version="4">
   <component name="CompilerConfiguration">
     <annotationProcessing>
+      <profile default="true" name="Default" enabled="true" />
```

Comment:
Why is this necessary?

Reply:
We should not be committing changes to files in .idea to our version control.

Reply:
done